### PR TITLE
Add hover effect to Cancel button in Advanced Filters

### LINKED ISSUE #4747: Add hover effect to cancel button in advance filter across platform
**Describe the bug**

Currently, the hover effect for buttons is inconsistent in the advance filters across the platform. the clear filter and apply buttons are having hover effect and the cancel button is not having a button hover effect ( across the patient tab, filter tab etc...)

**To Reproduce**

https://care.coronasafe.in/facility

**Expected behavior**

Add a hover effect for the cancel button in the advance filters of the facility page

**Screenshots**

image

### PR DESCRIPTION
**Pull request by Ashesh3**
Fixes #4747 

GenerousFantail

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

File: src/style/CAREUI.css [hunk 1]
```diff
@@ -102,8 +102,8 @@
 .button-primary-ghost { @apply accent-primary-500 hover:bg-primary-100 text-primary-500 !important }
 .button-primary-border { @apply border border-primary-500 }
 
-.button-secondary-default { @apply accent-secondary-200 bg-white hover:bg-secondary-200 text-secondary-800 !important }
-.button-secondary-ghost { @apply accent-secondary-200 hover:bg-secondary-100 text-secondary-700 !important }
+.button-secondary-default { @apply accent-secondary-300 bg-secondary-300 hover:bg-secondary-200 text-secondary-800 !important }
```
Comment (rithviknishad) [inline src/style/CAREUI.css]: @Ashesh3 this was intentionally kept white. How about reverting the changes for the default type, but preserving your changes for the ghost type alone?

```suggestion
.button-secondary-default { @apply accent-secondary-300 bg-secondary-300 hover:bg-secondary-200 text-secondary-800 !important }
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Preserve white bg for default variant

**Comment by Ashesh3** [comment]
> Preserve white bg for default variant

Done!

GenerousFantail

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**